### DISCUSSION TITLE
Github markdown

### DISCUSSION THREAD

**Discussion by mimpushkarev** [mdx-editor/editor#211] [Ideas]
Github markdown have a lot of features like "collapsed section", "tasks list" or "color model" that doesn't supported at common MD, are u thinking to create more plugins that can support like this one?

Collapsed sections: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections#creating-a-collapsed-section
Task lists: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists#about-task-lists
Color model: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#supported-color-models

**Comment by petyosi** [comment]
Thanks for sharing that - Markdown has a lot of flavors and options, that's the reason why I see the editor as a base with plugins model. Some thoughts on the above:

- Tasks (or todo lists) are already supported. 
- Details/summary are already supported (it's just an HTML), but you may want to design a custom toolbar UI (like a dialog) or even a special editor. Not so hard, you have the nested editors available. 
- Not sure how much it makes sense to support the colors, maybe a picker of some sort can be done.

**Comment by jaxomlotus** [comment]
One Github feature that would be wonderful to add is a way to translate @ users like Github does it (eg. @petyosi  )

**Comment by braincore** [comment]
@petyosi What level of sponsorship would you request to implement a customizable "@users" (with inline auto-complete) feature?

Happy to start a separate thread about it, but thought I'd ask here since @jaxomlotus did.

**Reply by petyosi** [reply]
@braincore I'm not yet totally certain on how to make such a feature generic enough. If you're interested, I'm happy to do a one-off work for your case - contact me at petyo@virtuoso.dev.

**Reply by jaxomlotus** [reply]
I'm interested as well.
